zenhappens/src/services.html

```diff
@@ -0,0 +1,371 @@
+---
+layout: layouts/default.njk
+title: Services | Zen Happens 
+---
+
+    {% include "_header.html" %}
+
+     <section class="services__hero">
+        <p class="eyebrow--onimage">Zen Happens in Lauraville</p>
+        <h3 class="landing__text">Services and Pricing</h3>
+    </section>
+    
+    <main class="services__main">
+    <section class="container">
+        <h2 class="price-list__title">Custom Massage</h2>
+        <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
```

Comment:
This seems appropriate to wrap in a `thead` (see [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead)):
```
<thead>
        <tr>
            <th colspan="2" class="u-sansserif"><strong>Regular Rates</strong></th>
        </tr>
<thead>
```
Note that the `td` also becomes a `th` here.